server/@types/ciagInductionApi/index.d.ts

```diff
@@ -4,6 +4,15 @@
  */
 
 export interface paths {
+  '/queue-admin/retry-dlq/{dlqName}': {
```

Comment:
I had to re-import the CIAG API spec which has brought in a load of noise which we dont care about ....